[JSC] Replace `typedef` with `using` for Modern C++ Style

Gumichocopengin8 opened the pull request:
#### 182872c4ce86011d6adf385550c8070e0dcb79e3
<pre>
[JSC] Replace `typedef` with `using` for Modern C++ Style
<a href="https://bugs.webkit.org/show_bug.cgi?id=292359">https://bugs.webkit.org/show_bug.cgi?id=292359</a>

Reviewed by NOBODY (OOPS!).

Replace `typedef` with `using` for Modern C++ Style

* Source/JavaScriptCore/assembler/ARM64Assembler.h:
* Source/JavaScriptCore/assembler/ARMv7Assembler.h:
* Source/JavaScriptCore/assembler/AbstractMacroAssembler.h:
* Source/JavaScriptCore/assembler/LinkBuffer.h:
* Source/JavaScriptCore/assembler/MacroAssembler.h:
* Source/JavaScriptCore/assembler/MacroAssemblerARM64.h:
* Source/JavaScriptCore/assembler/MacroAssemblerARMv7.h:
* Source/JavaScriptCore/assembler/MacroAssemblerX86_64.h:
* Source/JavaScriptCore/assembler/Printer.h:
* Source/JavaScriptCore/assembler/ProbeStack.h:
* Source/JavaScriptCore/assembler/X86Assembler.h:
* Source/JavaScriptCore/assembler/testmasm.cpp:
* Source/JavaScriptCore/b3/B3BasicBlock.h:
* Source/JavaScriptCore/b3/B3BlockInsertionSet.h:
* Source/JavaScriptCore/b3/B3BlockWorklist.h:
* Source/JavaScriptCore/b3/B3CFG.h:
* Source/JavaScriptCore/b3/B3CaseCollection.h:
* Source/JavaScriptCore/b3/B3ConstPtrValue.h:
* Source/JavaScriptCore/b3/B3EliminateCommonSubexpressions.cpp:
* Source/JavaScriptCore/b3/B3FrequentedBlock.h:
* Source/JavaScriptCore/b3/B3GenericBlockInsertionSet.h:
* Source/JavaScriptCore/b3/B3HeapRange.h:
* Source/JavaScriptCore/b3/B3InsertionSet.h:
* Source/JavaScriptCore/b3/B3NativeTraits.h:
* Source/JavaScriptCore/b3/B3NaturalLoops.h:
* Source/JavaScriptCore/b3/B3PatchpointValue.h:
* Source/JavaScriptCore/b3/B3PhiChildren.h:
* Source/JavaScriptCore/b3/B3Procedure.h:
* Source/JavaScriptCore/b3/B3PureCSE.h:
* Source/JavaScriptCore/b3/B3SparseCollection.h:
* Source/JavaScriptCore/b3/B3StackmapValue.h:
* Source/JavaScriptCore/b3/B3Value.h:
* Source/JavaScriptCore/b3/B3VariableLiveness.h:
* Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackByLinearScan.cpp:
* Source/JavaScriptCore/b3/air/AirBasicBlock.h:
* Source/JavaScriptCore/b3/air/AirBlockInsertionSet.h:
* Source/JavaScriptCore/b3/air/AirBlockWorklist.h:
* Source/JavaScriptCore/b3/air/AirCFG.h:
* Source/JavaScriptCore/b3/air/AirFrequentedBlock.h:
* Source/JavaScriptCore/b3/air/AirInsertionSet.h:
* Source/JavaScriptCore/b3/air/AirInst.h:
* Source/JavaScriptCore/b3/air/AirLiveness.h:
* Source/JavaScriptCore/b3/air/AirLivenessAdapter.h:
* Source/JavaScriptCore/b3/air/AirPrintSpecial.h:
* Source/JavaScriptCore/b3/air/AirRegLiveness.h:
* Source/JavaScriptCore/b3/testb3.h:
* Source/JavaScriptCore/bytecode/ArrayProfile.h:
* Source/JavaScriptCore/bytecode/BytecodeGeneratorification.cpp:
* Source/JavaScriptCore/bytecode/BytecodeGraph.h:
* Source/JavaScriptCore/bytecode/CallEdge.h:
* Source/JavaScriptCore/bytecode/CallVariant.h:
* Source/JavaScriptCore/bytecode/CodeBlock.h:
* Source/JavaScriptCore/bytecode/DirectEvalCodeCache.h:
* Source/JavaScriptCore/bytecode/EvalCodeBlock.h:
* Source/JavaScriptCore/bytecode/ExecutionCounter.h:
* Source/JavaScriptCore/bytecode/FunctionCodeBlock.h:
* Source/JavaScriptCore/bytecode/GlobalCodeBlock.h:
* Source/JavaScriptCore/bytecode/ICStatusMap.h:
* Source/JavaScriptCore/bytecode/InlineCallFrameSet.h:
* Source/JavaScriptCore/bytecode/ModuleProgramCodeBlock.h:
* Source/JavaScriptCore/bytecode/Opcode.h:
* Source/JavaScriptCore/bytecode/ProgramCodeBlock.h:
* Source/JavaScriptCore/bytecode/PutByStatus.h:
* Source/JavaScriptCore/bytecode/SpeculatedType.h:
* Source/JavaScriptCore/bytecode/TypeLocation.h:
* Source/JavaScriptCore/bytecode/UnlinkedCodeBlock.h:
* Source/JavaScriptCore/bytecode/UnlinkedEvalCodeBlock.h:
* Source/JavaScriptCore/bytecode/UnlinkedFunctionCodeBlock.h:
* Source/JavaScriptCore/bytecode/UnlinkedFunctionExecutable.h:
* Source/JavaScriptCore/bytecode/UnlinkedGlobalCodeBlock.h:
* Source/JavaScriptCore/bytecode/UnlinkedModuleProgramCodeBlock.h:
* Source/JavaScriptCore/bytecode/UnlinkedProgramCodeBlock.h:
* Source/JavaScriptCore/bytecompiler/BytecodeGenerator.h:
* Source/JavaScriptCore/bytecompiler/StaticPropertyAnalysis.h:
* Source/JavaScriptCore/bytecompiler/StaticPropertyAnalyzer.h:
* Source/JavaScriptCore/debugger/DebuggerParseData.cpp:
(JSC::gatherDebuggerParseData):
* Source/JavaScriptCore/dfg/DFGAdaptiveInferredPropertyValueWatchpoint.h:
* Source/JavaScriptCore/dfg/DFGBasicBlock.h:
* Source/JavaScriptCore/dfg/DFGBlockInsertionSet.h:
* Source/JavaScriptCore/dfg/DFGBlockWorklist.h:
* Source/JavaScriptCore/dfg/DFGCFG.h:
* Source/JavaScriptCore/dfg/DFGCSEPhase.cpp:
* Source/JavaScriptCore/dfg/DFGCommon.h:
* Source/JavaScriptCore/dfg/DFGDesiredWatchpoints.h:
* Source/JavaScriptCore/dfg/DFGInsertionSet.h:
* Source/JavaScriptCore/dfg/DFGIntegerRangeOptimizationPhase.cpp:
* Source/JavaScriptCore/dfg/DFGLivenessAnalysisPhase.cpp:
* Source/JavaScriptCore/dfg/DFGNode.h:
* Source/JavaScriptCore/dfg/DFGNodeFlags.h:
* Source/JavaScriptCore/dfg/DFGPhiChildren.h:
* Source/JavaScriptCore/dfg/DFGPureValue.h:
* Source/JavaScriptCore/dfg/DFGRegisterBank.h:
* Source/JavaScriptCore/dfg/DFGSpeculativeJIT.h:
* Source/JavaScriptCore/dfg/DFGTransition.h:
* Source/JavaScriptCore/ftl/FTLAbbreviatedTypes.h:
* Source/JavaScriptCore/ftl/FTLAbstractHeap.h:
* Source/JavaScriptCore/ftl/FTLLazySlowPath.h:
* Source/JavaScriptCore/ftl/FTLStackmapArgumentList.h:
* Source/JavaScriptCore/ftl/FTLThunks.h:
* Source/JavaScriptCore/heap/CellContainer.h:
* Source/JavaScriptCore/heap/DeferGC.h:
* Source/JavaScriptCore/heap/GCLogging.h:
* Source/JavaScriptCore/heap/GCSegmentedArray.h:
* Source/JavaScriptCore/heap/Handle.h:
* Source/JavaScriptCore/heap/HandleSet.h:
* Source/JavaScriptCore/heap/HandleTypes.h:
* Source/JavaScriptCore/heap/Heap.h:
* Source/JavaScriptCore/heap/HeapSnapshotBuilder.h:
* Source/JavaScriptCore/heap/MarkedBlock.h:
* Source/JavaScriptCore/heap/MarkedSpace.h:
* Source/JavaScriptCore/heap/SlotVisitor.h:
* Source/JavaScriptCore/heap/Strong.h:
* Source/JavaScriptCore/heap/Weak.h:
* Source/JavaScriptCore/inspector/InspectorBackendDispatcher.h:
* Source/JavaScriptCore/inspector/remote/RemoteConnectionToTarget.h:
* Source/JavaScriptCore/inspector/remote/RemoteInspector.h:
* Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorServer.h:
* Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorUtils.h:
* Source/JavaScriptCore/interpreter/Interpreter.h:
* Source/JavaScriptCore/jit/ExecutableMemoryHandle.h:
* Source/JavaScriptCore/jit/FPRInfo.h:
* Source/JavaScriptCore/jit/GPRInfo.h:
* Source/JavaScriptCore/jit/JITMathIC.h:
* Source/JavaScriptCore/jit/JITMathICForwards.h:
* Source/JavaScriptCore/jit/JITOperations.h:
* Source/JavaScriptCore/jit/Snippet.h:
* Source/JavaScriptCore/llint/LLIntData.h:
* Source/JavaScriptCore/llint/LLIntThunks.h:
* Source/JavaScriptCore/parser/ASTBuilder.h:
* Source/JavaScriptCore/parser/ModuleScopeData.h:
* Source/JavaScriptCore/parser/Nodes.h:
* Source/JavaScriptCore/parser/Parser.h:
* Source/JavaScriptCore/parser/ParserArena.h:
* Source/JavaScriptCore/parser/ParserModes.h:
* Source/JavaScriptCore/parser/SyntaxChecker.h:
* Source/JavaScriptCore/parser/VariableEnvironment.h:
* Source/JavaScriptCore/runtime/AbstractModuleRecord.h:
* Source/JavaScriptCore/runtime/ArgList.h:
* Source/JavaScriptCore/runtime/ArrayConstructor.h:
* Source/JavaScriptCore/runtime/AsyncFunctionConstructor.h:
* Source/JavaScriptCore/runtime/BasicBlockLocation.h:
* Source/JavaScriptCore/runtime/BrandedStructure.h:
* Source/JavaScriptCore/runtime/CachedTypes.cpp:
* Source/JavaScriptCore/runtime/CodeCache.cpp:
(JSC::generateUnlinkedCodeBlockImpl):
* Source/JavaScriptCore/runtime/CodeCache.h:
* Source/JavaScriptCore/runtime/ControlFlowProfiler.h:
* Source/JavaScriptCore/runtime/DateConstructor.h:
* Source/JavaScriptCore/runtime/FunctionConstructor.h:
* Source/JavaScriptCore/runtime/FunctionExecutable.h:
* Source/JavaScriptCore/runtime/FunctionPrototype.h:
* Source/JavaScriptCore/runtime/FunctionRareData.h:
* Source/JavaScriptCore/runtime/GeneratorFunctionConstructor.h:
* Source/JavaScriptCore/runtime/GetPutInfo.h:
* Source/JavaScriptCore/runtime/Identifier.h:
* Source/JavaScriptCore/runtime/IndexingType.h:
* Source/JavaScriptCore/runtime/IntlCollatorConstructor.h:
* Source/JavaScriptCore/runtime/IntlDateTimeFormatConstructor.h:
* Source/JavaScriptCore/runtime/IntlNumberFormatConstructor.h:
* Source/JavaScriptCore/runtime/IntlPluralRulesConstructor.h:
* Source/JavaScriptCore/runtime/JSArray.h:
* Source/JavaScriptCore/runtime/JSAsyncFunction.h:
* Source/JavaScriptCore/runtime/JSCJSValue.h:
* Source/JavaScriptCore/runtime/JSCustomGetterFunction.h:
* Source/JavaScriptCore/runtime/JSCustomSetterFunction.h:
* Source/JavaScriptCore/runtime/JSFunction.h:
* Source/JavaScriptCore/runtime/JSGenericTypedArrayView.h:
* Source/JavaScriptCore/runtime/JSInternalPromise.h:
* Source/JavaScriptCore/runtime/JSInternalPromisePrototype.h:
* Source/JavaScriptCore/runtime/JSIteratorConstructor.h:
* Source/JavaScriptCore/runtime/JSModuleNamespaceObject.h:
* Source/JavaScriptCore/runtime/JSObject.h:
* Source/JavaScriptCore/runtime/JSString.h:
* Source/JavaScriptCore/runtime/JSTypeInfo.h:
* Source/JavaScriptCore/runtime/JSTypedArrayPrototypes.h:
* Source/JavaScriptCore/runtime/LazyClassStructure.h:
* Source/JavaScriptCore/runtime/LazyProperty.h:
* Source/JavaScriptCore/runtime/MapConstructor.h:
* Source/JavaScriptCore/runtime/NativeExecutable.h:
* Source/JavaScriptCore/runtime/NullGetterFunction.h:
* Source/JavaScriptCore/runtime/NullSetterFunction.h:
* Source/JavaScriptCore/runtime/NumberPrototype.cpp:
* Source/JavaScriptCore/runtime/ObjectConstructor.h:
* Source/JavaScriptCore/runtime/PropertyNameArray.h:
* Source/JavaScriptCore/runtime/PropertyOffset.h:
* Source/JavaScriptCore/runtime/PropertyStorage.h:
* Source/JavaScriptCore/runtime/ProxyConstructor.h:
* Source/JavaScriptCore/runtime/ProxyRevoke.h:
* Source/JavaScriptCore/runtime/RegExpCache.h:
* Source/JavaScriptCore/runtime/RegExpConstructor.h:
* Source/JavaScriptCore/runtime/RuntimeType.h:
* Source/JavaScriptCore/runtime/SamplingProfiler.cpp:
* Source/JavaScriptCore/runtime/ScopedArgumentsTable.h:
* Source/JavaScriptCore/runtime/ScriptExecutable.h:
* Source/JavaScriptCore/runtime/SetConstructor.h:
* Source/JavaScriptCore/runtime/ShadowRealmConstructor.h:
* Source/JavaScriptCore/runtime/SparseArrayValueMap.h:
* Source/JavaScriptCore/runtime/Structure.h:
* Source/JavaScriptCore/runtime/StructureCache.h:
* Source/JavaScriptCore/runtime/StructureRareData.cpp:
* Source/JavaScriptCore/runtime/StructureRareData.h:
* Source/JavaScriptCore/runtime/StructureTransitionTable.h:
* Source/JavaScriptCore/runtime/Symbol.h:
* Source/JavaScriptCore/runtime/SymbolConstructor.h:
* Source/JavaScriptCore/runtime/SymbolTable.h:
* Source/JavaScriptCore/runtime/TemplateObjectDescriptor.h:
* Source/JavaScriptCore/runtime/TypeProfiler.h:
* Source/JavaScriptCore/runtime/TypedArrayAdaptersForwardDeclarations.h:
* Source/JavaScriptCore/runtime/TypedArrayAdaptors.h:
* Source/JavaScriptCore/runtime/VM.h:
* Source/JavaScriptCore/runtime/WeakGCMap.h:
* Source/JavaScriptCore/runtime/WeakMapConstructor.h:
* Source/JavaScriptCore/runtime/WeakSetConstructor.h:
* Source/JavaScriptCore/tools/CellList.h:
* Source/JavaScriptCore/tools/Integrity.h:
* Source/JavaScriptCore/tools/JSDollarVM.cpp:
* Source/JavaScriptCore/tools/JSDollarVM.h:
* Source/JavaScriptCore/tools/ProfileTreeNode.h:
* Source/JavaScriptCore/wasm/WasmFormat.h:
* Source/JavaScriptCore/wasm/WasmIndexOrName.h:
* Source/JavaScriptCore/wasm/WasmOMGIRGenerator.cpp:
* Source/JavaScriptCore/wasm/WasmOMGIRGenerator32_64.cpp:
* Source/JavaScriptCore/wasm/WasmOperations.h:
* Source/JavaScriptCore/wasm/WasmParser.h:
* Source/JavaScriptCore/wasm/WasmWorklist.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyCompileErrorConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyGCStructure.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyInstanceConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyLinkErrorConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyMemoryConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyModuleConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyRuntimeErrorConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyTableConstructor.h:
* Source/JavaScriptCore/yarr/YarrJIT.cpp:
* Source/JavaScriptCore/yarr/YarrParser.h:
</pre><!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/182872c4ce86011d6adf385550c8070e0dcb79e3

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/104037 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/23741 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/14061 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/109233 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/54704 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/106077 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/24103 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/32285 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/79028 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/107043 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/18715 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/93841 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/59357 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/18510 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/11892 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/54065 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/96712 "Built successfully and passed tests") | [❌ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/88235 "1 api test failed or timed out") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/11950 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/111618 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/102648 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/31193 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/22987 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/88041 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/31557 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/90041 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/87698 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/32581 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/10328 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/25625 "The change is no longer eligible for processing.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/31122 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/36434 "Built successfully") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/126281 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/30915 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/34924 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/34252 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/32476 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

Constellation commented on Source/JavaScriptCore/API/JSRemoteInspector.h:
```diff
@@ -31,10 +31,10 @@
 #include <unistd.h>
-typedef pid_t JSProcessID;
+using JSProcessID = pid_t;
 #endif
```
They are API headers. So They need to be interpreted as C (not C++).

Gumichocopengin8 committed:
[JSC] Replace `typedef` with `using` for Modern C++ Style
https://bugs.webkit.org/show_bug.cgi?id=292359

Reviewed by NOBODY (OOPS!).

Replace `typedef` with `using` for Modern C++ Style

* Source/JavaScriptCore/assembler/ARM64Assembler.h:
* Source/JavaScriptCore/assembler/ARMv7Assembler.h:
* Source/JavaScriptCore/assembler/AbstractMacroAssembler.h:
* Source/JavaScriptCore/assembler/LinkBuffer.h:
* Source/JavaScriptCore/assembler/MacroAssembler.h:
* Source/JavaScriptCore/assembler/MacroAssemblerARM64.h:
* Source/JavaScriptCore/assembler/MacroAssemblerARMv7.h:
* Source/JavaScriptCore/assembler/MacroAssemblerX86_64.h:
* Source/JavaScriptCore/assembler/Printer.h:
* Source/JavaScriptCore/assembler/ProbeStack.h:
* Source/JavaScriptCore/assembler/X86Assembler.h:
* Source/JavaScriptCore/assembler/testmasm.cpp:
* Source/JavaScriptCore/b3/B3BasicBlock.h:
* Source/JavaScriptCore/b3/B3BlockInsertionSet.h:
* Source/JavaScriptCore/b3/B3BlockWorklist.h:
* Source/JavaScriptCore/b3/B3CFG.h:
* Source/JavaScriptCore/b3/B3CaseCollection.h:
* Source/JavaScriptCore/b3/B3ConstPtrValue.h:
* Source/JavaScriptCore/b3/B3EliminateCommonSubexpressions.cpp:
* Source/JavaScriptCore/b3/B3FrequentedBlock.h:
* Source/JavaScriptCore/b3/B3GenericBlockInsertionSet.h:
* Source/JavaScriptCore/b3/B3HeapRange.h:
* Source/JavaScriptCore/b3/B3InsertionSet.h:
* Source/JavaScriptCore/b3/B3NativeTraits.h:
* Source/JavaScriptCore/b3/B3NaturalLoops.h:
* Source/JavaScriptCore/b3/B3PatchpointValue.h:
* Source/JavaScriptCore/b3/B3PhiChildren.h:
* Source/JavaScriptCore/b3/B3Procedure.h:
* Source/JavaScriptCore/b3/B3PureCSE.h:
* Source/JavaScriptCore/b3/B3SparseCollection.h:
* Source/JavaScriptCore/b3/B3StackmapValue.h:
* Source/JavaScriptCore/b3/B3Value.h:
* Source/JavaScriptCore/b3/B3VariableLiveness.h:
* Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackByLinearScan.cpp:
* Source/JavaScriptCore/b3/air/AirBasicBlock.h:
* Source/JavaScriptCore/b3/air/AirBlockInsertionSet.h:
* Source/JavaScriptCore/b3/air/AirBlockWorklist.h:
* Source/JavaScriptCore/b3/air/AirCFG.h:
* Source/JavaScriptCore/b3/air/AirFrequentedBlock.h:
* Source/JavaScriptCore/b3/air/AirInsertionSet.h:
* Source/JavaScriptCore/b3/air/AirInst.h:
* Source/JavaScriptCore/b3/air/AirLiveness.h:
* Source/JavaScriptCore/b3/air/AirLivenessAdapter.h:
* Source/JavaScriptCore/b3/air/AirPrintSpecial.h:
* Source/JavaScriptCore/b3/air/AirRegLiveness.h:
* Source/JavaScriptCore/b3/testb3.h:
* Source/JavaScriptCore/bytecode/ArrayProfile.h:
* Source/JavaScriptCore/bytecode/BytecodeGeneratorification.cpp:
* Source/JavaScriptCore/bytecode/BytecodeGraph.h:
* Source/JavaScriptCore/bytecode/CallEdge.h:
* Source/JavaScriptCore/bytecode/CallVariant.h:
* Source/JavaScriptCore/bytecode/CodeBlock.h:
* Source/JavaScriptCore/bytecode/DirectEvalCodeCache.h:
* Source/JavaScriptCore/bytecode/EvalCodeBlock.h:
* Source/JavaScriptCore/bytecode/ExecutionCounter.h:
* Source/JavaScriptCore/bytecode/FunctionCodeBlock.h:
* Source/JavaScriptCore/bytecode/GlobalCodeBlock.h:
* Source/JavaScriptCore/bytecode/ICStatusMap.h:
* Source/JavaScriptCore/bytecode/InlineCallFrameSet.h:
* Source/JavaScriptCore/bytecode/ModuleProgramCodeBlock.h:
* Source/JavaScriptCore/bytecode/Opcode.h:
* Source/JavaScriptCore/bytecode/ProgramCodeBlock.h:
* Source/JavaScriptCore/bytecode/PutByStatus.h:
* Source/JavaScriptCore/bytecode/SpeculatedType.h:
* Source/JavaScriptCore/bytecode/TypeLocation.h:
* Source/JavaScriptCore/bytecode/UnlinkedCodeBlock.h:
* Source/JavaScriptCore/bytecode/UnlinkedEvalCodeBlock.h:
* Source/JavaScriptCore/bytecode/UnlinkedFunctionCodeBlock.h:
* Source/JavaScriptCore/bytecode/UnlinkedFunctionExecutable.h:
* Source/JavaScriptCore/bytecode/UnlinkedGlobalCodeBlock.h:
* Source/JavaScriptCore/bytecode/UnlinkedModuleProgramCodeBlock.h:
* Source/JavaScriptCore/bytecode/UnlinkedProgramCodeBlock.h:
* Source/JavaScriptCore/bytecompiler/BytecodeGenerator.h:
* Source/JavaScriptCore/bytecompiler/StaticPropertyAnalysis.h:
* Source/JavaScriptCore/bytecompiler/StaticPropertyAnalyzer.h:
* Source/JavaScriptCore/debugger/DebuggerParseData.cpp:
(JSC::gatherDebuggerParseData):
* Source/JavaScriptCore/dfg/DFGAdaptiveInferredPropertyValueWatchpoint.h:
* Source/JavaScriptCore/dfg/DFGBasicBlock.h:
* Source/JavaScriptCore/dfg/DFGBlockInsertionSet.h:
* Source/JavaScriptCore/dfg/DFGBlockWorklist.h:
* Source/JavaScriptCore/dfg/DFGCFG.h:
* Source/JavaScriptCore/dfg/DFGCSEPhase.cpp:
* Source/JavaScriptCore/dfg/DFGCommon.h:
* Source/JavaScriptCore/dfg/DFGDesiredWatchpoints.h:
* Source/JavaScriptCore/dfg/DFGInsertionSet.h:
* Source/JavaScriptCore/dfg/DFGIntegerRangeOptimizationPhase.cpp:
* Source/JavaScriptCore/dfg/DFGLivenessAnalysisPhase.cpp:
* Source/JavaScriptCore/dfg/DFGNode.h:
* Source/JavaScriptCore/dfg/DFGNodeFlags.h:
* Source/JavaScriptCore/dfg/DFGPhiChildren.h:
* Source/JavaScriptCore/dfg/DFGPureValue.h:
* Source/JavaScriptCore/dfg/DFGRegisterBank.h:
* Source/JavaScriptCore/dfg/DFGSpeculativeJIT.h:
* Source/JavaScriptCore/dfg/DFGTransition.h:
* Source/JavaScriptCore/ftl/FTLAbbreviatedTypes.h:
* Source/JavaScriptCore/ftl/FTLAbstractHeap.h:
* Source/JavaScriptCore/ftl/FTLLazySlowPath.h:
* Source/JavaScriptCore/ftl/FTLStackmapArgumentList.h:
* Source/JavaScriptCore/ftl/FTLThunks.h:
* Source/JavaScriptCore/heap/CellContainer.h:
* Source/JavaScriptCore/heap/DeferGC.h:
* Source/JavaScriptCore/heap/GCLogging.h:
* Source/JavaScriptCore/heap/GCSegmentedArray.h:
* Source/JavaScriptCore/heap/Handle.h:
* Source/JavaScriptCore/heap/HandleSet.h:
* Source/JavaScriptCore/heap/HandleTypes.h:
* Source/JavaScriptCore/heap/Heap.h:
* Source/JavaScriptCore/heap/HeapSnapshotBuilder.h:
* Source/JavaScriptCore/heap/MarkedBlock.h:
* Source/JavaScriptCore/heap/MarkedSpace.h:
* Source/JavaScriptCore/heap/SlotVisitor.h:
* Source/JavaScriptCore/heap/Strong.h:
* Source/JavaScriptCore/heap/Weak.h:
* Source/JavaScriptCore/inspector/InspectorBackendDispatcher.h:
* Source/JavaScriptCore/inspector/remote/RemoteConnectionToTarget.h:
* Source/JavaScriptCore/inspector/remote/RemoteInspector.h:
* Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorServer.h:
* Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorUtils.h:
* Source/JavaScriptCore/interpreter/Interpreter.h:
* Source/JavaScriptCore/jit/ExecutableMemoryHandle.h:
* Source/JavaScriptCore/jit/FPRInfo.h:
* Source/JavaScriptCore/jit/GPRInfo.h:
* Source/JavaScriptCore/jit/JITMathIC.h:
* Source/JavaScriptCore/jit/JITMathICForwards.h:
* Source/JavaScriptCore/jit/JITOperations.h:
* Source/JavaScriptCore/jit/Snippet.h:
* Source/JavaScriptCore/llint/LLIntData.h:
* Source/JavaScriptCore/llint/LLIntThunks.h:
* Source/JavaScriptCore/parser/ASTBuilder.h:
* Source/JavaScriptCore/parser/ModuleScopeData.h:
* Source/JavaScriptCore/parser/Nodes.h:
* Source/JavaScriptCore/parser/Parser.h:
* Source/JavaScriptCore/parser/ParserArena.h:
* Source/JavaScriptCore/parser/ParserModes.h:
* Source/JavaScriptCore/parser/SyntaxChecker.h:
* Source/JavaScriptCore/parser/VariableEnvironment.h:
* Source/JavaScriptCore/runtime/AbstractModuleRecord.h:
* Source/JavaScriptCore/runtime/ArgList.h:
* Source/JavaScriptCore/runtime/ArrayConstructor.h:
* Source/JavaScriptCore/runtime/AsyncFunctionConstructor.h:
* Source/JavaScriptCore/runtime/BasicBlockLocation.h:
* Source/JavaScriptCore/runtime/BrandedStructure.h:
* Source/JavaScriptCore/runtime/CachedTypes.cpp:
* Source/JavaScriptCore/runtime/CodeCache.cpp:
(JSC::generateUnlinkedCodeBlockImpl):
* Source/JavaScriptCore/runtime/CodeCache.h:
* Source/JavaScriptCore/runtime/ControlFlowProfiler.h:
* Source/JavaScriptCore/runtime/DateConstructor.h:
* Source/JavaScriptCore/runtime/FunctionConstructor.h:
* Source/JavaScriptCore/runtime/FunctionExecutable.h:
* Source/JavaScriptCore/runtime/FunctionPrototype.h:
* Source/JavaScriptCore/runtime/FunctionRareData.h:
* Source/JavaScriptCore/runtime/GeneratorFunctionConstructor.h:
* Source/JavaScriptCore/runtime/GetPutInfo.h:
* Source/JavaScriptCore/runtime/Identifier.h:
* Source/JavaScriptCore/runtime/IndexingType.h:
* Source/JavaScriptCore/runtime/IntlCollatorConstructor.h:
* Source/JavaScriptCore/runtime/IntlDateTimeFormatConstructor.h:
* Source/JavaScriptCore/runtime/IntlNumberFormatConstructor.h:
* Source/JavaScriptCore/runtime/IntlPluralRulesConstructor.h:
* Source/JavaScriptCore/runtime/JSArray.h:
* Source/JavaScriptCore/runtime/JSAsyncFunction.h:
* Source/JavaScriptCore/runtime/JSCJSValue.h:
* Source/JavaScriptCore/runtime/JSCustomGetterFunction.h:
* Source/JavaScriptCore/runtime/JSCustomSetterFunction.h:
* Source/JavaScriptCore/runtime/JSFunction.h:
* Source/JavaScriptCore/runtime/JSGenericTypedArrayView.h:
* Source/JavaScriptCore/runtime/JSInternalPromise.h:
* Source/JavaScriptCore/runtime/JSInternalPromisePrototype.h:
* Source/JavaScriptCore/runtime/JSIteratorConstructor.h:
* Source/JavaScriptCore/runtime/JSModuleNamespaceObject.h:
* Source/JavaScriptCore/runtime/JSObject.h:
* Source/JavaScriptCore/runtime/JSString.h:
* Source/JavaScriptCore/runtime/JSTypeInfo.h:
* Source/JavaScriptCore/runtime/JSTypedArrayPrototypes.h:
* Source/JavaScriptCore/runtime/LazyClassStructure.h:
* Source/JavaScriptCore/runtime/LazyProperty.h:
* Source/JavaScriptCore/runtime/MapConstructor.h:
* Source/JavaScriptCore/runtime/NativeExecutable.h:
* Source/JavaScriptCore/runtime/NullGetterFunction.h:
* Source/JavaScriptCore/runtime/NullSetterFunction.h:
* Source/JavaScriptCore/runtime/NumberPrototype.cpp:
* Source/JavaScriptCore/runtime/ObjectConstructor.h:
* Source/JavaScriptCore/runtime/PropertyNameArray.h:
* Source/JavaScriptCore/runtime/PropertyOffset.h:
* Source/JavaScriptCore/runtime/PropertyStorage.h:
* Source/JavaScriptCore/runtime/ProxyConstructor.h:
* Source/JavaScriptCore/runtime/ProxyRevoke.h:
* Source/JavaScriptCore/runtime/RegExpCache.h:
* Source/JavaScriptCore/runtime/RegExpConstructor.h:
* Source/JavaScriptCore/runtime/RuntimeType.h:
* Source/JavaScriptCore/runtime/SamplingProfiler.cpp:
* Source/JavaScriptCore/runtime/ScopedArgumentsTable.h:
* Source/JavaScriptCore/runtime/ScriptExecutable.h:
* Source/JavaScriptCore/runtime/SetConstructor.h:
* Source/JavaScriptCore/runtime/ShadowRealmConstructor.h:
* Source/JavaScriptCore/runtime/SparseArrayValueMap.h:
* Source/JavaScriptCore/runtime/Structure.h:
* Source/JavaScriptCore/runtime/StructureCache.h:
* Source/JavaScriptCore/runtime/StructureRareData.cpp:
* Source/JavaScriptCore/runtime/StructureRareData.h:
* Source/JavaScriptCore/runtime/StructureTransitionTable.h:
* Source/JavaScriptCore/runtime/Symbol.h:
* Source/JavaScriptCore/runtime/SymbolConstructor.h:
* Source/JavaScriptCore/runtime/SymbolTable.h:
* Source/JavaScriptCore/runtime/TemplateObjectDescriptor.h:
* Source/JavaScriptCore/runtime/TypeProfiler.h:
* Source/JavaScriptCore/runtime/TypedArrayAdaptersForwardDeclarations.h:
* Source/JavaScriptCore/runtime/TypedArrayAdaptors.h:
* Source/JavaScriptCore/runtime/VM.h:
* Source/JavaScriptCore/runtime/WeakGCMap.h:
* Source/JavaScriptCore/runtime/WeakMapConstructor.h:
* Source/JavaScriptCore/runtime/WeakSetConstructor.h:
* Source/JavaScriptCore/tools/CellList.h:
* Source/JavaScriptCore/tools/Integrity.h:
* Source/JavaScriptCore/tools/JSDollarVM.cpp:
* Source/JavaScriptCore/tools/JSDollarVM.h:
* Source/JavaScriptCore/tools/ProfileTreeNode.h:
* Source/JavaScriptCore/wasm/WasmFormat.h:
* Source/JavaScriptCore/wasm/WasmIndexOrName.h:
* Source/JavaScriptCore/wasm/WasmOMGIRGenerator.cpp:
* Source/JavaScriptCore/wasm/WasmOMGIRGenerator32_64.cpp:
* Source/JavaScriptCore/wasm/WasmOperations.h:
* Source/JavaScriptCore/wasm/WasmParser.h:
* Source/JavaScriptCore/wasm/WasmWorklist.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyCompileErrorConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyGCStructure.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyInstanceConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyLinkErrorConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyMemoryConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyModuleConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyRuntimeErrorConstructor.h:
* Source/JavaScriptCore/wasm/js/WebAssemblyTableConstructor.h:
* Source/JavaScriptCore/yarr/YarrJIT.cpp:
* Source/JavaScriptCore/yarr/YarrParser.h:
